.gemini-cli/INSTALL.md

```
@@ -0,0 +1,137 @@
+
+✅ **14+ skills** covering:
+- TDD and testing
+- Debugging and investigation
+- Planning and design
+- Collaboration and code review
+- Meta-skills for creating new skills
+
+✅ **Automatic skill activation** - Skills load when relevant
+
+✅ **Persistent context** - Superpowers guidance in every session
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)